src/tests/ftest/util/dmg_utils.py

```diff
@@ -51,13 +51,14 @@ class DmgCommand(YamlCommand):
         # UUID since it's made up of hyphens and \s includes new line.
         "pool_list": r"(?:([0-9a-fA-F-]+) +([0-9,]+))",
         "pool_create": r"(?:UUID:|Service replicas:)\s+([A-Za-z0-9-]+)",
-        "pool_query": r"(?:Pool\s*([A-Za-z0-9-]+),\s*ntarget=([0-9]),"
-                      r"\s*disabled=([0-9]),\s+leader=([0-9]),"
-                      r"\s+version=([0-9])|Target\(VOS\) count:\s*([0-9])|"
+        "pool_query": r"(?:Pool\s+([0-9a-fA-F-]+),\s+ntarget=(\d+),"
+                      r"\s+disabled=(\d+),\s+leader=(\d+),"
+                      r"\s+version=(\d+)|Target\(VOS\)\s+count:\s*(\d+)|"
                       r"(?:(?:SCM:|NVMe:)\s+Total\s+size:\s+([0-9.]+\s+[A-Z]+)"
                       r"\s+Free:\s+([0-9.]+\s+[A-Z]+),\smin:([0-9.]+\s+[A-Z]+)"
                       r",\s+max:([0-9.]+\s+[A-Z]+),\s+mean:([0-9.]+\s+[A-Z]+))"
-                      r"|Rebuild\s+idle,\s+([0-9]+)\s+objs,\s+([0-9]+)\s+recs)"
+                      r"|Rebuild\s+\w+,\s+([0-9]+)\s+objs,\s+([0-9]+)"
```

Comment:
Why not create a new capture group for the rebuild state by adding the parenthesis? Like so:
`r"|Rebuild\s+(\w+),\s+([0-9]+)\s+objs,\s+([0-9]+)"`

Reply:
I am not planning to create a new capture group now... It is not required for my script. Atleast based on the different stages, it doesn't seem to be useful for the present query... Also, Adding rebuild status to present regex causes pool query tests fail.. So not planning this change now. Will leave it to some other testers work based on requirement.